src/routes/meteors.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const { getAsteroidsData } = require('../controllers/asteroids');
+const { getAsteroidsData } = require('../controllers/asteroids.js');
```

Comment:
Seems like ../controllers/asteroids should be enough (here & below)

Reply:
Thank you. Updated.